benchopt/tests/test_benchmarks/dummy_benchmark/doc_objective.py

```diff
@@ -0,0 +1,26 @@
+from benchopt.base import BaseObjective
+
+
+class Objective(BaseObjective):
+    name = "Lasso Regression"
+
+    parameters = {
+        'reg': [0.05, .1, .5]
+    }
+
+    def __init__(self, reg=.1):
+        self.reg = reg
+
+    def set_data(self, X, y):
+        self.X, self.y = X, y
+        self.lmbd = self.reg * self._get_lambda_max()
+
+    def compute(self, beta):
+        diff = self.y - self.X.dot(beta)
+        return .5 * diff.dot(diff) + self.lmbd * abs(beta).sum()
+
+    def _get_lambda_max(self):
+        return abs(self.X.T.dot(self.y)).max()
+
+    def to_dict(self):
+        return dict(X=self.X, y=self.y, lmbd=self.lmbd)
```

Comment:
As this version is for the doc, I think we could add a couple of comments here no?

Reply:
Comments added for compute, to_dict and parameters